tests/test_plugin.py

```diff
@@ -33,3 +33,26 @@ def test_destroy(fxa_client, fxa_account):
     result = testdir.runpytest()
     result.assert_outcomes(error=1, passed=1)
     assert 'ClientError: Unknown account' in result.stdout.str()
+
+
+def test_commandline_email_option(testdir):
+    testdir.makepyfile("""
+        import pytest
+
+        def test_account(fxa_account):
+            assert 'testemail@restmail.net' in fxa_account.email
+    """)
+    result = testdir.runpytest('--fxa-email', 'testemail@restmail.net')
+    result.assert_outcomes(passed=1)
+
+
+def test_fxa_email_env_variable(testdir, monkeypatch):
+    monkeypatch.setenv('FXA_EMAIL', 'testemail@restmail.net')
```

Comment:
These tests are failing in Travis, and I suspect it's because they're all using the  same email address. Can we generate a random address for the purposes of testing?